packages/react/src/dialog/root/DialogRoot.test.tsx

```diff
@@ -388,4 +390,186 @@ describe('<Dialog.Root />', () => {
 
     expect(notifyTransitionEnd.callCount).to.equal(1);
   });
+
+  describe('prop: modal', () => {
+    it('should render an internal backdrop when `true`', async () => {
```

Comment:
These two tests are identical.

Reply:
Thanks for spotting, you're right. Seems like the Menu and Select ones are different but this one wasn't changed.